quirks/password-rules.json

```diff
@@ -251,6 +251,9 @@
     "fedex.com": {
         "password-rules": "minlength: 8; max-consecutive: 3; required: lower; required: upper; required: digit; allowed: [-!@#$%^&*_+=`|(){}[:;,.?]];"
     },
+    "flysas.com": {
+        "password-rules": "minlength: 8; maxlength: 14; required: lower; required: upper; required: digit; required: [(-~!@#$%^&_+=`|(){}[:\"'<>,.?]];"
```

Comment:
In a character class, `-` needs to be the first character, if present.

Reply:
Thank you, fixed!